client/src/Homes.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styles from './style.css';
+
+const Homes = (props) => {
+  const { data } = props;
+  const listing = data.map((home) => {
+    return (
```

Comment:
Can you take out the curly braces and return statement here to make this a single-line return?